remote.rb

```diff
@@ -10,16 +10,16 @@
 data = []
 
 page.css('.owwb-cs-slide-list-account-item').each do |i|
-  account = i.css('.contract-number').text
-  balance = i.css('.owwb-cs-slide-list-amount-value').text
-  currency = i.css('.owwb-cs-slide-list-amount-currency').text
-  nature = i.css('.owwb-cs-slide-list-properties-list-property-value').text.gsub('[MDL]', '').strip
-  
-  data.push(  
-    name: account,
-    balance: balance,
-    currency: currency ,
-    nature: nature
+    account  = i.css('.contract-number').text
+    balance  = i.css('.owwb-cs-slide-list-amount-value').text
+    currency = i.css('.owwb-cs-slide-list-amount-currency').text
+    #nature = 
```

Comment:
try not to push in master commented code. if it's commented means it's useless. you can write this when you'll be in need